src/com/javarush/task/task08/task0816/Solution.java

```diff
@@ -0,0 +1,96 @@
+package com.javarush.task.task08.task0816;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+/*
+Добрая Зинаида и летние каникулы
+*/
+
+public class Solution {
+    public static Map<String, Date> createMap() throws ParseException {
+        DateFormat dateFormat = new SimpleDateFormat("MMMMM d yyyy", Locale.ENGLISH);
+        Map<String, Date> map = new HashMap<>();
+        map.put("Смирнов", dateFormat.parse("MAY 1 2012"));
+
+        //напишите тут ваш код
+        map.put("Смирнов1", dateFormat.parse("JANUARY 1 2012"));
+        map.put("Смирнов2", dateFormat.parse("JANUARY 1 2012"));
+        map.put("Смирнов3", dateFormat.parse("APRIL 1 2012"));
+        map.put("Смирнов4", dateFormat.parse("JANUARY 1 2012"));
+        map.put("Смирнов5", dateFormat.parse("FEBRUARY 1 2012"));
+        map.put("Смирнов6", dateFormat.parse("MARCH 1 2012"));
+        map.put("Смирнов7", dateFormat.parse("JULY 1 2012"));
+        map.put("Смирнов8", dateFormat.parse("DECEMBER 1 2012"));
+        map.put("Смирнов9", dateFormat.parse("SEPTEMBER 1 2012"));
+        return map;
+    }
+
+    public static void removeAllSummerPeople(Map<String, Date> map) {
+        //напишите тут ваш код
+        Map<String, Date> mapa = new HashMap<>();
+        Iterator iterator = map.entrySet().iterator();
+        while (iterator.hasNext()) {
+            Map.Entry entry = (Map.Entry) iterator.next();
+            Date date = (Date) entry.getValue();
+            if ((date.getMonth() != Calendar.JULY) && (date.getMonth() != Calendar.JUNE) && (date.getMonth() != Calendar.AUGUST)) {
+                String s = (String) entry.getKey();
+                mapa.put(s, date);
+            }
+        }
+        map = mapa;
+        iterator = map.entrySet().iterator();
+        while (iterator.hasNext()) {
+            Map.Entry entry = (Map.Entry) iterator.next();
+            System.out.println(entry.getValue());
+        }
```

Comment:
Я понял про какую дичь ты говорил.
```suggestion
                map.values().removeIf(date -> date.getMonth() == 5 || date.getMonth() == 6 || date.getMonth() == 7);
```